Cloud Application Manager/Deploying Anywhere/using-microsoft-azure.md

```diff
@@ -295,18 +295,24 @@ Azure ARM Templates are supported on Cloud Application Manager with Microsoft Az
    * Choose a name and description for the new instance.
    * Press **Deploy**.
 
+
 ### Registering Existing Instances from your Azure Account
 
 You can import existing Virtual Machines into you workspace only in one click. The list of available instances you can import come with your Microsoft Azure provider synchronization.
 
 #### Available Instances
 
-As part of the result of synchronization process you can find a list of available virtual machines that already exist in your account but not used yet in Cloud Application Manager. You can import an existing one clicking **Import** button.
+As part of the result of synchronization process you can find a list of available virtual machines or scalesets that already exist in your account but not used yet in Cloud Application Manager. You can import an existing one clicking **Import** button.
 
 ![Microsoft Azure - Available instances](../../images/cloud-application-manager/microsoft-azure/available-instances-9.png)
 
 You should first synchronize your Azure provider before trying to register the virtual machine in order to get the current available instances that can be registered along with their statuses.
 
+#### Managed Services Anywhere Enabled Providers
+
+If you are using a Managed Services Anywhere enabled provider in the deployment, once you deploy an ARM template, all the created instances (VMs and VM ScaleSets) will be automatically registered and linked from the UI. 
```

Comment:
```suggestion
If you are deploying an ARM template box into a Managed Services Anywhere enabled provider, all the created instances (VMs and VM ScaleSets) will be automatically registered and linked with the ARM instance that deployed them. In these registered instances, not all lifecycle actions will be allowed, since for them you will need to act on the parent ARM instance for the dependant linked instances to be updated.
```